packages/sdk/src/react/hooks/hooks.ts

```
@@ -0,0 +1,119 @@
+        const fetchData = useCallback(async () => {
+            // Wait until lock is released
+            while (isUpdating.current) {
+                await sleep(50);
+            }
+
+            // Lock function
+            isUpdating.current = true;
+
+            if (subscriptionRef.current) {
+                const [results, clause] = await config.queryToHashedKeysMethod(
+                    fetchingRef.current!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)